docs/src/main/asciidoc/secret-management.adoc

```diff
@@ -116,26 +120,55 @@ spring:
 @SpringBootApplication
 public class SampleApplication implements CommandLineRunner {
 
-    @Value("${secret-name-in-key-vault-1}")
-    private String secretNameInKeyVault1;
-    @Value("${secret-name-in-key-vault-2}")
-    private String secretNameInKeyVault2;
-    @Value("${secret-name-in-key-vault-both}")
-    private String secretNameInKeyVaultBoth;
+    @Value("${sampleProperty1}")
```

Comment:
Why do you change the names here, if you are willing to do this change, will you also update the names in samples？
![image](https://user-images.githubusercontent.com/4465723/148915764-9720373e-3011-4e26-b52a-7f85ab7345a4.png)

Reply:
PR created: https://github.com/Azure-Samples/azure-spring-boot-samples/pull/151